packages/signaling-server/src/index.js

```diff
@@ -1,16 +1,21 @@
 const socketIO = require('socket.io')
+const redis = require('socket.io-redis')
 
 const createRoomPool = require('./createRoomPool')
 const Node = require('./Node')
 
-function SignalingServer () {
+function SignalingServer ({ redisHost, redisPort }) {
```

Comment:
I would group the properties, something like

```js
const config = {
   redis: {
     host: '...',
     port: 4747
   }
}

const server = SignalingServer(config)
```

Reply:
Much better 👍